src/sentry/api/endpoints/organization_details.py

```diff
@@ -50,32 +52,40 @@ def update_organization_scenario(runner):
         )
 
 
-class OrganizationSerializer(serializers.ModelSerializer):
-    accountRateLimit = serializers.IntegerField(min_value=0, max_value=1000000)
-    projectRateLimit = serializers.IntegerField(min_value=50, max_value=100)
-    slug = serializers.RegexField(r'^[a-z0-9_\-]+$', max_length=50,
-                                  required=False)
-    avatar = AvatarField()
+class OrganizationSerializer(serializers.Serializer):
+    name = serializers.CharField(max_length=64)
+    slug = serializers.RegexField(r'^[a-z0-9_\-]+$', max_length=50)
+    accountRateLimit = serializers.IntegerField(min_value=0, max_value=1000000,
+                                                required=False)
+    projectRateLimit = serializers.IntegerField(min_value=50, max_value=100,
+                                                required=False)
+    avatar = AvatarField(required=False)
     avatarType = serializers.ChoiceField(choices=(
         ('upload', 'upload'),
         ('letter_avatar', 'letter_avatar'),
-    ))
-
-    class Meta:
-        model = Organization
-        fields = ('name', 'slug')
+    ), required=False)
+
+    openMembership = serializers.BooleanField(required=False)
+    allowSharedIssues = serializers.BooleanField(required=False)
+    enhancedPrivacy = serializers.BooleanField(required=False)
+    dataScrubber = serializers.BooleanField(required=False)
+    dataScrubberDefaults = serializers.BooleanField(required=False)
+    sensitiveFields = ListField(child=serializers.CharField(), required=False)
+    safeFields = ListField(child=serializers.CharField(), required=False)
+    scrubIPAddresses = serializers.BooleanField(required=False)
```

Comment:
I was curious if we had a precedent for naming `IP` in all caps or camel/capital-case (`Ip`).

The `user` interface `get_api_context` writes `ip_address` as `ipAddress` (for API consumption), not `IPAddress`. Not sure if this changes anything for you.